course_catalog/etl/podcast.py

```diff
@@ -127,7 +127,7 @@ def transform_episode(rss_data, offered_by, topics, parent_image):
         dict:
             normalized podcast episode data
     """
-    episode_id = generate_unique_id(rss_data.enclosure["url"])
+    episode_id = generate_unique_id(rss_data.guid.text)
```

Comment:
This change seems to create many duplicate episodes in the database, one with the old id and one with the new id (episode count after running backpopulate on master: 1297; after running again on this branch: 2413).  But the older ones get unpublished so it's not apparent to the end user.  My initial feeling was that the episodes with old id's should be deleted before importing again, but on second thought they don't seem to be doing any obvious harm.  What do you think?

Reply:
As you said, those unpublished episodes will be ignored. I think it makes sense to manually delete all unpublished episodes from the database after this is merged just to avoid clutter in the database